openedx/core/djangoapps/content_libraries/signals.py

```diff
@@ -2,16 +2,13 @@
 Content libraries related signals.
 """
 
-from django.dispatch import Signal
+# Re-export signals from openedx_events.content_authoring.signals
 
-# providing_args=['library_key']
-CONTENT_LIBRARY_CREATED = Signal()
-# providing_args=['library_key', 'update_blocks']
-CONTENT_LIBRARY_UPDATED = Signal()
-# providing_args=['library_key']
-CONTENT_LIBRARY_DELETED = Signal()
-
-# Same providing_args=['library_key', 'usage_key'] for next 3 signals.
-LIBRARY_BLOCK_CREATED = Signal()
-LIBRARY_BLOCK_DELETED = Signal()
-LIBRARY_BLOCK_UPDATED = Signal()
+# from openedx_events.content_authoring.signals import (
+#     CONTENT_LIBRARY_CREATED as CONTENT_LIBRARY_CREATED,
+#     CONTENT_LIBRARY_UPDATED as CONTENT_LIBRARY_UPDATED,
+#     CONTENT_LIBRARY_DELETED as CONTENT_LIBRARY_DELETED,
+#     LIBRARY_BLOCK_CREATED as LIBRARY_BLOCK_CREATED,
+#     LIBRARY_BLOCK_UPDATED as LIBRARY_BLOCK_UPDATED,
+#     LIBRARY_BLOCK_DELETED as LIBRARY_BLOCK_DELETED,
+# )
```

Comment:
This file can be deleted since we're using the signals from openedx-events now.

Anyone who might have been importing these events will need to update their code to use the new caller/receiver structure anyway, so better to delete this file than to import the new events and pretend they're the same.